### ISSUE TITLE
isomorphic

### ISSUE THREAD

**Issue by colinmegill** [FormidableLabs/recipes-flux#9]
Almost every implementation of flux has a thread on their issues page about making this better. I haven't gotten into it yet. 

Here's the reflux discussion: https://github.com/spoike/refluxjs/issues/144
Here's the fluxxor discussion: https://github.com/BinaryMuse/fluxxor/issues/5

Super young stuff. Again, it's in the realm of possibility that we write the defining flux implementation ourselves and OSS alongside Radium and React-HTTP (if we decide there's enough there to release a package). 

Long story short, I don't have a plan for this yet.

**Comment by ryan-roemer** [comment]
Fully-featured webpack for building isomorphic apps. I _think_ this enables webpack-style aliases to work in Node (which normally doesn't work). Kind of complicated (and in CoffeeScript), but worth a study -- https://gist.github.com/lettertwo/e35b7a57151b65a80c1d

**Comment by colinmegill** [comment]
This just got easier with the McFly integration - @kenwheeler could you post a couple of those reference links you showed me here? Maybe to files in the postocracy repo?

**Comment by kenwheeler** [comment]
In this project, I was using actions to load data into stores prior to an isomorphic render:

See the 'catchall' route. I define the api routes and then anything else uses React-Router
https://github.com/kenwheeler/react-postocracy/blob/master/config/routes.js

I set up an API module to fetch data(or mocks) and load them using Actions:
https://github.com/kenwheeler/react-postocracy/blob/master/api/Api.js

However, now that McFly is promise based, you can hit the API from an action like:
http://jsfiddle.net/thekenwheeler/32hgqsxt/
